custom/icds_reports/management/commands/delete_users_for_state.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.users.models import CommCareUser
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'user_list_file'
+        )
+        parser.add_argument(
+            'site_code'
+        )
+
+    def handle(self, user_list_file, site_code, *args, **kwargs):
+
+        with open(user_list_file) as fin:
+            users_to_delete = fin.readlines()
+            users_to_delete = [user.strip() for user in users_to_delete]
+            choice = raw_input('Total {} users will be deleted: (Y/N)'.format(len(users_to_delete)))
+            if not choice.startswith('y'):
+                print("Nice!!! Its good to be safe than to be sorry")
+                return
+
+            print("Deleting users")
+            users_submitted_forms = list()
+            for user in users_to_delete:
+                user_name = user + '@icds-cas.commcarehq.org'
+                mobile_user = CommCareUser.get_by_username(user_name)
```

Comment:
This will return `None` for any users that have already been deleted, causing the next line to fail.